include/fiction/algorithms/network_transformation/node_duplication_planarization.hpp

```diff
@@ -0,0 +1,794 @@
+//
+// Created by benjamin on 11.06.24.
+//
+
+#ifndef FICTION_NODE_DUPLICATION_PLANARIZATION_HPP
+#define FICTION_NODE_DUPLICATION_PLANARIZATION_HPP
+
+#include "fiction/algorithms/graph/mincross.hpp"
+#include "fiction/algorithms/network_transformation/network_balancing.hpp"
+#include "fiction/networks/virtual_pi_network.hpp"
+
+#include <mockturtle/traits.hpp>
+#include <mockturtle/utils/node_map.hpp>
+
+#include <algorithm>
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <cstdint>
+#include <limits>
+#include <random>
+#include <stdexcept>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+namespace fiction
+{
+
+/**
+ * Parameters for the node duplication algorithm.
+ */
+struct node_duplication_planarization_params
+{
+    /**
+     * The output order determines the starting layer for this algorithm. If this option is turned off, the output order
+     * remains the same as in the provided network. If it is turned on, the outputs are ordered randomly.
+     */
+    enum class output_order : uint8_t
```

Comment:
The docstring doesn't match the `enum class` content. There is no `ON` and `OFF` enumerators.